src/Polynomials.jl

```diff
@@ -348,14 +348,24 @@ end
 div(num::Poly, den::Poly) = divrem(num, den)[1]
 rem(num::Poly, den::Poly) = divrem(num, den)[2]
 
-function ==(p1::Poly, p2::Poly)
-    if p1.var != p2.var
-        return false
-    else
-        return p1.a == p2.a
-    end
+==(p1::Poly, p2::Poly) = (p1.var == p2.var && p1.a == p2.a)
+==(p1::Poly, n::Number) = (coeffs(p1) == [n])
+==(n::Number, p1::Poly) = (p1 == n)
+
+function isapprox{T,S}(p1::Poly{T}, p2::Poly{S};
+  rtol::Real = sqrt(eps(promote_type(T,S))), atol::Real = 0)
```

Comment:
I would like to remove the `Base.eps` overload. 

In `gcd` we have this check: `reduce(&, (@compat abs.(b.a)) .<=2*eps(S))`. Which requires defining `eps(Integer)`. I'd much rather see something like `b ≈ zero(S)` there. This would even work with Integer types, as `rtoldefaults(Integer, Integer) = 0`.